core/src/test/scala/unit/kafka/server/OffsetFetchRequestTest.scala

```diff
@@ -138,9 +137,9 @@ class OffsetFetchRequestTest extends BaseRequestTest {
     groupToPartitionMap.put(groups(3), null)
     groupToPartitionMap.put(groups(4), null)
 
-    createTopic(topic1)
-    createTopic(topic2, numPartitions = 2)
-    createTopic(topic3, numPartitions = 3)
+    createTopic(topics(0))
+    createTopic(topics(1), numPartitions = 2)
+    createTopic(topics(2), numPartitions = 3)
 
     val topicOneOffsets = topic1List.asScala.map{
```

Comment:
The changes suggested in the following method below to work on the collection can also be applied to `testOffsetFetchRequestWithMultipleGroups`.